docs/modules/ROOT/images/sequence.puml

```diff
@@ -1,12 +1,12 @@
 @startuml
```

Comment:
@patilsuraj767  - will it be possible for you to generate image using latest `sequence.puml`? Also, you would need to update yuptoo arch. diagram.

Do you want to address these changes into same PR?

Reply:
Yes, I will update the images